charts/gateway/templates/configmap.yaml

```diff
@@ -18,13 +18,21 @@ metadata:
 {{- end }}
 data:
   ACCEPT_LICENSE: {{ .Values.license.accept | quote}}
+  DISKLESS_CONFIG: {{ .Values.disklessConfig.enabled | quote }}
   SSG_CLUSTER_HOST: {{ .Values.clusterHostname }}
   SSG_JVM_HEAP: {{ .Values.config.heapSize }}
-{{- if .Values.database.enabled }}
-  {{- if .Values.database.create }}
+  {{- if and (.Values.config.minHeapSize) (.Values.config.maxHeapSize) }}
+  SSG_JVM_MIN_HEAP: {{ .Values.config.minHeapSize }}
+  SSG_JVM_MAX_HEAP: {{ .Values.config.maxHeapSize }}
+  {{- end }}
+{{- if .Values.disklessConfig.enabled }}
+  {{- if .Values.database.enabled }}
+  LIQUIBASE_LOG_LEVEL: {{ default "off" .Values.database.liquibaseLogLevel | quote }}
```

Comment:
LIQUIBASE_LOG_LEVEL is applicable regardless of disklessConfig.enabled setting. The if statements should be reorganized. Right now, it won't be set if disklessConfig.enabled=false.

Reply:
Updated